src/test/kotlin/unit/service/UserServiceTest.kt

```diff
@@ -0,0 +1,42 @@
+//package unit.service
+//
+//import com.epam.kotlinapp.crud.business.*
+//import com.epam.kotlinapp.crud.dao.*
+//import com.epam.kotlinapp.crud.dao.nosql.*
+//import com.epam.kotlinapp.crud.model.*
+//import org.junit.*
+//import org.junit.Test
+//import org.junit.runner.*
+//import org.mockito.*
+//import org.mockito.BDDMockito.*
+//import org.mockito.junit.*
+//import kotlin.test.*
+//
+////@RunWith(value = MockitoJUnitRunner::class)
+//class UserServiceTest {
```

Comment:
it should be deleted or add comments

Reply:
fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6